tonic/functional/to_frame.py

```diff
@@ -66,9 +66,12 @@ def to_frame_numpy(
         events["p"] = 0
 
     if time_window:
-        event_slices = slice_events_by_time(
-            events, time_window, overlap=overlap, include_incomplete=include_incomplete
-        )
+        if len(events) == 0:
```

Comment:
the check for the case that an empty list of events is passed should probably happen a bit higher up, for example here:
https://github.com/neuromorphs/tonic/blob/7455d0084710c95a4a6d56eed7f0c07cfaccd238/tonic/transforms.py#L894

Reply:
Check is now performed in ToFrame transform. Not sure about the behavior, though. The modified implementation is the one that makes most sense to me.